### PR TITLE
Elaborate Task.status

### LINKED ISSUE #3: Define status and state attributes in the SHOP4CF JSONLD context
We should define `status` attribute of Task, as discussed in [that comment thread](https://github.com/SHOP4CF/data-models/pull/2#discussion_r658697273).

### PR DESCRIPTION
**Pull request by michzimny**

### CODE REVIEW

File: docs/task.md [hunk 1]
```diff
@@ -94,6 +94,12 @@ Further notes on the attributes:
 - `locationFunction` within `happensAt` is optional, an its values are use-case specific
 - `workParameters` are use-case specific and are set by the party that orders the task
 - `status` represents the current execution status with the following allowed values:
-`pending`, `assigned`, `inProgress`, `completed`, `paused`, `suspended`, `failed`
+  - `pending` - execution of this Task is requested, but the component (the agent) responsible for its execution is not yet aware
```
Comment (matteopantano) [inline docs/task.md:97]: This description is good for my side. Regarding suspended I would wait for @kontrag

**Review by matteopantano — COMMENTED** [review]
Reviewed

File: docs/task.md (hunk 1 above)
Comment (kontrag) [reply docs/task.md:97]: Literally speaking, paused and suspended have slightly different meaning but in practice I'm not sure what is the differentiating point. The way I typically see it is that pause is a break after which the activity resumes, while suspend is halting the activity but the resume might not happen (e.g. one decides to completely "delete"/"fail" the activity). I've seen cases where Suspend is used when there's a safety risk (e.g. a human in front of a robot) and the activity is "stopped" (if there's no other way to change the robot's behavior and continue the process).

At the moment, I cannot think of a generic definition that would clearly distinguish and require both values, so maybe we can keep only one (e.g. paused) and see in the future if we need to add the other.

Comment (michzimny) [reply docs/task.md:97]: Ok, removing `suspended`.

**Pull request merged**